migrate-to-chimera

```diff
@@ -79,11 +79,18 @@ if [ -d $HOME/.local/share/steam-shortcuts ] ; then
             echo "Renaming steam-buddy.${p}.yaml to chimera.${p}.yaml"
             mv -Tbv "steam-buddy.${p}.yaml" "chimera.${p}.yaml"
             echo "Renamig paths inside chimera.${p}.yaml"
-            sed -i 'si|/steam-buddy/|/chimera/|g' "chimera.${p}.yaml"
+            sed -i 's|/steam-buddy/|/chimera/|g' "chimera.${p}.yaml"
```

Comment:
I don't think we need this anymore.

Reply:
You mean use the next statement to catch them all?